Automattic/newspack-theme
# feat: redesign "My Account" page

### All Submissions:






### Changes proposed in this Pull Request:

Implements a column style for WC's "My Account" page on desktop.

<img width="1216" alt="image" src="https://user-images.githubusercontent.com/820752/178782194-c08c748e-fcee-4d87-9d40-b2e1a9971fed.png">

The link background color should be updated to use the `primary` global variable, which is being tackled at #1875 

Closes #1878 

### How to test the changes in this Pull Request:

1. Make sure your "My Account" page uses the "One column" template
2. Visit WC's "My Account" page while logged out
3. Confirm the login page style is unaltered
4. Authenticate and confirm the new UI
5. Change your viewport to mobile (< 768) and confirm the navigation is responsive

### Other information:
